PULL_REQUEST_TEMPLATE_md

```diff
@@ -0,0 +1,17 @@
+### All Submissions:
+
+* [ ] Have you followed the guidelines in our Contributing document?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
```

Comment:
Since this is intended to be read by the PR author while they are writing their PR message, where the Markdown will not be rendered (unless they click the "Preview" tab), I think providing a complete URL makes things more clear. I have also modified the URL so the search results will show closed as well as open PRs (since they should search both to see if the proposal was already rejected).
```suggestion
* [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/arduino/Arduino/pulls?q=) for the same update/change?
```